README.md

```
@@ -66,6 +66,9 @@ will simply update the comfy.yaml file to reflect the local setup
   pip install comfyui-manager==4.1b8          # specific version
   ```
 - `comfy --workspace=<path> install`: Install ComfyUI into `<path>/ComfyUI`.
+- `comfy install --fast-deps`: Use `uv` instead of `pip` for faster dependency resolution 
```

markdownlint-cli2 (0.22.0)

[warning] Trailing spaces
Expected: 0 or 2; Actual: 1

(MD009, no-trailing-spaces)